lib/manageiq/providers/workflows/runner.rb

```diff
@@ -0,0 +1,91 @@
+module ManageIQ
+  module Providers
+    module Workflows
+      class Runner
+        include Vmdb::Logging
+
+        class << self
+          def runner
+            @runner ||= new.tap(&:start)
+          end
```

Comment:
NOTE this ensures that this will work from the AutomationWorker as well as in development using simulate_queue_worker.